app/src/components/v-upload.vue

```diff
@@ -55,10 +58,42 @@ onUnmounted(() => {
 });
 
 function validFiles(files: FileList) {
-	if (files.length === 0) return false;
-
 	for (const file of files) {
-		if (file.size === 0) return false;
+		if (file.size === 0) {
```

Comment:
I'm wondering if we should be explicit and show an error when trying to upload a non-image as part of the set, rather than ignoring the non-images and continuing the upload for the others 🤔